pages/database-management/monitoring.mdx

```diff
@@ -85,48 +137,122 @@ connection is dropped. As a response, the following message is sent:
 }
 ```
 
-#### Authorization (Enterprise)
+To use WebSocket to authenticate, send the JSON message to the WebSocket server:
 
-Permission for connecting through WebSocket is controlled by the [`WEBSOCKET`
-privilege](/database-management/authentication-and-authorization/role-based-access-control#privileges).
+```Python
+import asyncio
+import json
+import websockets
 
-### Log messages
+HOST = "localhost"
+PORT = "7444"
+USERNAME = "memgraph"
+PASSWORD = "memgraph"
 
-Each log that is written to the log file is forwarded to the connected clients
-in the following format:
+ws_url = f"ws://{HOST}:{PORT}"
 
-```json
-{
-  event: "log",
-  level: "trace"|"debug"|"info"|"warning"|"error"|"critical",
-  message: "<log-message>"
-}
+
+async def authenticate_and_read_logs(websocket):
+    # Send authentication credentials
+ credentials = json.dumps({"username": USERNAME, "password": PASSWORD})
+    await websocket.send(credentials)
+
+    # Wait for authentication response
+ response = await websocket.recv()
+ response_data = json.loads(response)
+    if response_data.get("success"):
+        print("Authentication successful!")
+    else:
+        print(f"Authentication failed: {response_data.get('message')}")
+        return  # Stop if authentication fails
+
+    # After successful authentication, start reading logs
+    try:
+        while True:
+ log_message = await websocket.recv()
+            print(log_message)
+    except websockets.exceptions.ConnectionClosed:
+        print("Connection to Memgraph closed.")
+
+
+async def connect_and_authenticate():
+    async with websockets.connect(ws_url) as websocket:
+        print(f"Connected to Memgraph at {ws_url}")
+        await authenticate_and_read_logs(websocket)
+
+
+asyncio.run(connect_and_authenticate())
 ```
 
-Read more about [logs](/database-management/logs) in Memgraph.
+
+
+#### Authorization (Enterprise)
```

Comment:
is it h4 on purpose?

Reply:
Yeah, it is related to the WEB socket auth access.